src/libraries/Microsoft.Extensions.Logging/tests/Common/ILLink.Descriptors.xml

```diff
@@ -0,0 +1,7 @@
+<linker>
+  <assembly fullname="System.Diagnostics.TraceSource">
```

Comment:
This one is a bit unexpected, do you know where is it used?

Reply:
Agreed - this shouldn't be necessary because all the ILLink warnings are fixed in this assembly. This might be a product bug somewhere.